Python/cp/LetterCount.py

```diff
@@ -0,0 +1,37 @@
+"""
+The program uses dictionary in python
+to count the number of occurence a letter
+makes in a word
+"""
+
+
+def LetterCount(word):
+    dict = {}
+
+    for c in word:
+        co = dict.get(c, 0)
+        if co == 0:
+            dict[c] = 1
+        else:
+            dict[c] = co + 1
+
+    return(dict)
+
+if __name__ == '__main__':
+    word = input("enter a word ").lower()
+    LetterCount = LetterCount(word)
+    print(LetterCount)
+
+else:
```

Comment:
I don't think we need an else statement here.

Reply:
@chicken-biryani This looks like an Anti-Pattern

Reply:
@atarax665 @HarshCasper Apologies. Part of learning I guess (Thank you for the new word, Anti-pattern).